source/math/FloatShell2D.ooc

```diff
@@ -28,9 +28,9 @@ FloatShell2D: cover {
 	isZero ::= this left == 0 && this right == 0 && this top == 0 && this bottom == 0
 	notZero ::= this left != 0 && this right != 0 && this top != 0 && this bottom != 0
 	init: func@ (=left, =right, =top, =bottom)
-	init: func@ ~fromFloat (value: Float) { this init(value, value) }
+	init: func@ ~fromFloat (value: Float) { this init(value, value, value, value) }
 	init: func@ ~fromFloats (x, y: Float) { this init(x, x, y, y) }
-	init: func@ ~default { this init(0.0f) }
+	init: func@ ~default { this init(0.0f, 0.0f, 0.0f, 0.0f) }
```

Comment:
This was actually on purpose (chained constructors), but I guess it doesn't matter either way.